.travis.yml

```diff
@@ -15,7 +15,6 @@ notifications:
       - gm130s@gmail.com  # Change this to yours should you copy this file.
 env:
   global:
-    - DEBUG_BASH='true' # Print bash trace
```

Comment:
I was not able to find the errors in the +x enabled code ;)
It easy to add for a certain test.